content/en/install.md

```diff
@@ -50,43 +55,87 @@ Other options include:
   and the IEP interactive development environment; Supports Linux,
   Windows, and Mac.
 
+If you already have Python installed, you can
+[install SciPy with `pip`](#pip-install).
+For more advanced users who will need to install or upgrade regularly,
+you may prefer to use a dedicated package manager like [`uv`]
+(see [Installing with `uv`](#uv-install)),
+[Conda], or [`pixi`] (see [Installing from conda-forge](#conda-forge-install)).
+
+[`uv`]: https://docs.astral.sh/uv/
+[Conda]: https://docs.conda.io/projects/conda/en/latest/index.html
+[`pixi`]: https://pixi.sh/latest/
+
+<a name="pypi-install"></a>
+
+## Installing from PyPI
+
+SciPy is [available on PyPI](https://pypi.org/project/scipy/), the Python Package Index.
+
 <a name="pip-install"></a>
 
-# Installing with Pip
+### Installing with `pip`
 
 You can install SciPy from PyPI with `pip`:
 
     python -m pip install scipy
 
-<a name="conda-install"></a>
+<a name="uv-install"></a>
+
+### Installing with `uv`
+
+You can also add SciPy from PyPI to a `uv` project:
+
+    uv add scipy
+
+<a name="conda-forge-install"></a>
 
-# Installing via Conda
+## Installing from conda-forge
 
-You can install SciPy from the `defaults` or `conda-forge` channels with `conda`:
+### Installing with `conda` or `mamba`
+
+[`Miniforge`] is the recommended way to install the `conda`
+(and `mamba`, a faster `conda` alternative) package manager.
```

Comment:
This is becoming less true as the faster engine got into conda. Maybe worth asking someone from the conda team.

Reply:
Good point, @jaimergp perhaps you could clarify whether it seems worthwhile to distinguish between `conda` and `mamba` here?

Reply:
If I understand correctly, (broadly speaking, from a non-expert user perspective)

- `conda` now provides the speed and effectively all of the benefits `mamba` used to have
- mamba 2.0 makes `mamba` very similar to `micromamba`, which is still more lightweight in a sense than `conda`?

Reply:
I'd mention conda and micromamba. Ultimately it's up to the user though, and their muscle memory. Mamba 2.0 is like micromamba but dynamically linked.

Reply:
I don't agree, `mamba` 2.0 still has the same UX as `mamba` 1.x - so like `conda` not like `minimamba`, see https://mamba.readthedocs.io/en/latest/user_guide/mamba.html#mamba-vs-conda-clis), it's just further improved. And it has features `conda` doesn't have (e.g. `repoquery` is super useful). CLI output is also better, and I think (but am not sure) that downloads are faster as well.

So I'd much prefer to keep `mamba`, as the reasoning for adding it hasn't really changed - it's still a better `conda`.

Reply:
You really have to separate thinking about this from a user perspective vs. from an internals perspective. Dynamic linking is irrelevant to a user, unless it somehow translates into observable differences in behavior.

Reply:
In which case, I'm inclined to leave this text as is. Given https://github.com/mamba-org/mamba/pull/3610, it seems that micromamba is intended to be more developer facing than user facing now. cc @jjerphan for visibility.

Reply:
In a nutshell:
 - `mamba` and `micromamba` are the same program, only their linkage change (`mamba` is dynamically linked, whilst `micromamba` is nearly totally-statically linked)
 - `conda` now uses `libmamba` to solve environments (via `conda-libmamba-solver`), so performance is on par with `mamba`
 - `mamba` and `conda` have differences in features
 - `mamba` is targeting conda-forge by default now

I would recommend using `mamba`.

See [the differences between `mamba` and `micromamba`](https://github.com/mamba-org/mamba/?tab=readme-ov-file#mamba-vs-micromamba).